docs-kits/kits/Circularity_KIT/page-adoption-view.md

```diff
@@ -551,97 +552,6 @@ The Return Request aspect model could be used to flag a vehicle or product to in
 
 The Product Passport aspect model could provide reliable details on the product characteristics or lifecycle information, therefore complementing the basic Marketplace Offer information. More information on this aspect model can be found under CX-0103 in the standards library or under the following link: [Digital Product Passport](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.generic.digital_product_passport)
```

Comment:
I'd suggest to update CX-0103 to "CX-0143 - Use Case Circular Economy Digital Product Passport Standard" as the successor; or alternatively, directly reference the current (R24.05) standardized version of the semantic model (v4.0): https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.generic.digital_product_passport/4.0.0